### DISCUSSION TITLE
🧩 FEATURE REQUEST: Add keyboard accessibility to the Home Page

### DISCUSSION THREAD

**Discussion by Prajwalprakash3722** [dailydotdev/daily#526] [Feature Requests]
## Feature Request
Add keyboard accessibility to the Home Page

### Description
I am a great Keyboard, shortcut lover, I use keyboard more instead of mouse.

As we know that when we press `ctrl` + `n` { n | [1,9]} we can go the respective tab and when we press `ctrl` + `Tab` we can alternate between the tabs. in the same way is it possible to have the accessibility to a story by pressing a particular set of keys so that we can use mouse less often. 

### Solution
Maybe we can start to listen to keyboard events and when a particular key stroke is applied we can activate this function.

### Alternate Solutions

Keep Pressing `Tab` until it selects a whole story as a link, which will take more key strokes as the home Page has more links and input fields.

**Comment by idoshamun** [comment]
Hi @Prajwalprakash3722,

We are keyboard shortcut lovers as well. Keyboard shortcuts are on the roadmap but do not have an ETA at the moment.
We'll keep you posted here

**Reply by Prajwalprakash3722** [reply]
Any updates on this, would love to work on this and contribute:)

**Reply by idoshamun** [reply]
Unfortunately no update. But let me see if we can get the ball rolling on our side so you can contribute
